config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```
@@ -88,7 +88,48 @@ periodics:
     description: Runs conformance tests using image against latest kubernetes master with a kubernetes-in-docker cluster
     testgrid-alert-email: davanum@gmail.com
     testgrid-num-columns-recent: '3'
-- interval: 1h
+- interval: 3h
+  name: ci-kubernetes-conformance-image-test-test-15
```

Still not matching the 1-14 job below
```suggestion
  name: ci-kubernetes-conformance-image-1-15-test
```